modules/optimeraBidAdapter.js

```diff
@@ -0,0 +1,75 @@
+import {registerBidder} from 'src/adapters/bidderFactory';
+const BIDDER_CODE = 'optimera';
+const SCORES_BASE_URL = 'https://s3.amazonaws.com/elasticbeanstalk-us-east-1-397719490216/json/client/';
+
+export const spec = {
+  code: BIDDER_CODE,
+  /**
+   * Determines whether or not the given bid request is valid.
+   *
+   * @param {bidRequest} bid The bid params to validate.
+   * @return boolean True if this is a valid bid, and false otherwise.
+   */
+  isBidRequestValid: function (bidRequest) {
+    return !!(bidRequest.params.custom.clientID);
```

Comment:
This will throw an exception `custom.clientID` is not present.  Should do safe evaluation, `isBidRequestValid` should not throw.